.devcontainer/devcontainer.json

```
@@ -1,33 +1,11 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.222.0/containers/ubuntu
 {
   "name": "Ubuntu",
   "build": {
-    "dockerfile": "Dockerfile",
-    // Update 'VARIANT' to pick an Ubuntu version: hirsute, focal, bionic
-    // Use hirsute or bionic on local arm64/Apple Silicon.
-    "args": { "VARIANT": "hirsute" }
+    "dockerfile": "Dockerfile"
   },
-
-  // Set *default* container specific settings.json values on container create.
-  "settings": {},
-
-
-  // Add the IDs of extensions you want installed when the container is created.
-  "extensions": [],
-
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  // "forwardPorts": [],
-
-  // Use 'postCreateCommand' to run commands after the container is created.
-  // "postCreateCommand": "uname -a",
-  "postCreateCommand": "bash ./.devcontainer/postCreateCommand.sh",
-
-  // Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-  "remoteUser": "vscode",
   "features": {
-    "git": "latest",
-    "node": "16",
-    "python": "3.10"
-  }
+    "ghcr.io/devcontainers/features/node:1": {},
+    "ghcr.io/devcontainers/features/python:1": {}
```

```suggestion
    "ghcr.io/devcontainers/features/git:1": {},
    "ghcr.io/devcontainers/features/git-lfs:1": {},
    "ghcr.io/devcontainers/features/github-cli:1": {},
    "ghcr.io/devcontainers/features/node:1": {}
```

If I recall correctly from my work in #378, the addition of Python here was causing the build to take a very long time and we can do without it by installing HTTPie like so: 

https://github.com/jessehouwing/training-manual/blob/91d87e1a28e0db12a1f0d601880c2d773c8d8f0c/.devcontainer/Dockerfile#L10-L12